packages/client/verifiable-credentials/src/presentation/getNfts.ts

```diff
@@ -3,44 +3,57 @@ import { Nft, isVcChain } from "@/verifiableCredentialsSDK";
 import { crossmintAPI } from "../crossmintAPI";
 import { CrossmintWalletNft } from "../types/nfts";
 
-export async function getWalletNfts(chain: string, wallet: string) {
+async function fetchPaginatedData(url: string): Promise<any[]> {
```

Comment:
I think this method can become useless if the response size is huge. One option would be paginating it and transfer this pagination param to the backend but this seems like the perfect case for using an AsyncGenerator https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/AsyncGenerator

Wdyt?

Reply:
I know if the response is huge our backend would crash even before but thinking on the long term if we solve the pagination problem with the NFTs fetch :D